packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -96,9 +93,15 @@ export const buildModules = async (
       },
       experimental: {
         renderBuiltUrl(filename, { type }) {
-          const domain = getModuleDomain(fileInfo.path);
+          let domain = `http://localhost:8000`;
+          if (typeof process.env.YEXT_SITE_ARGUMENT !== "undefined") {
+            domain = new URL(
+              "http://" +
+                JSON.parse(process.env.YEXT_SITE_ARGUMENT).productionDomain
```

Comment:
I think we need to be careful here in the case the parse fails and handle that. Also, I think this should be https. It might work with just // which means "use whatever protocol the user is on" but you'd have to test that out.

Reply:
Added try / catch. Tested w/ // but didn't work so am using https